apps/new/widget/page/project/Main.jsx

```diff
@@ -26,7 +26,10 @@ const { Layout } = VM.require(
   Layout: () => <></>,
 };
 const config = {
-  theme: {},
+  theme: {
+    "--font-family": "Poppins, sans-serif",
+    fontFamily: "var(--font-family)",
+  },
```

Comment:
Do these get used anywhere?

Reply:
they are applied globally to the project page, was really trying to fix it.